assignment1.txt

```diff
@@ -0,0 +1,7 @@
+1. The 'return 0' at the end of the code tells the compiler that the code is done. Since main is initialized as an int, it needs the 0 to tell the compiler it can stop compiling.
```

Comment:
Not only does it tell the compiler that it's done, but that it has 0 errors - the rest of your answer is right!